package Exceptions;

public class Use_of_Throws {
public static void main(String[] args) 
	throws Throwable{
		System.out.println("hii");
		Thread.sleep(2000);
	
}
}
